content/docs/command-reference/update.md

```diff
@@ -78,14 +75,12 @@ compatible when updating from a worktree remote.
 - `--no-download` - Update data checksums in the `.dvc` file (`md5`, `etag`, or
   `checksum` fields) without actually downloading the latest data. See
   `dvc import-url --no-download` or `dvc import --no-download` for more context.
-  Cannot be combined with `--to-remote`.
+  Cannot be used with `--to-remote` or [worktree updates](#worktree-update).
```

Comment:
Blocking: Support for this is being worked on in https://github.com/iterative/dvc/issues/8653, so let's not document it as unsupported 🙏

Reply:
Sure, it was just a copy edit of what's already published but we can remove this if you prefer. Feel free to commit @dberenbaum :
```suggestion
```

Reply:
```suggestion
  Cannot be used with `--to-remote`.
```

Since `worktree` is no longer supported, let's just drop the reference to it.